docs/source/welcome/services/index.md

```diff
@@ -0,0 +1,64 @@
+(servicesprovided)=
+# Services we Provide
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`device-desktop;1.5em;screen-full` Compute
+:link: ../../hardware/hardware_overview
+:link-type: doc
+
+An overview of our computational resources.
++++
+[Learn more »](../../hardware/hardware_overview)
+:::
+
+:::{grid-item-card} {octicon}`database;1.5em;sd-mr-1` Storage
+:link: ../../datamanagement/discovery_storage
+:link-type: doc
+
+We offer several storage solutions for active and archival use.
++++
+[Learn more »](../../datamanagement/discovery_storage)
+:::
+
+:::{grid-item-card} {octicon}`mortar-board;1.5em;sd-mr-1` Classroom
+:link: ../../classroom/class_use
+:link-type: doc
+
+We support classroom use of HPC resources.
```

Comment:
```suggestion
Thinking about using HPC resources in your course, review the courses FAQ.
```
We should probably try to stick to using 'course' rather than 'classroom' for consistency and clarity.